### DISCUSSION TITLE
Is there any way to make a dataloader return Flux instead of a CompletionStage<List> ?

### DISCUSSION THREAD

**Discussion by ghost** [Netflix/dgs-framework#858] [Q&A] [answered]
I am doing some dataloaders in a reactive environment using webflux.
I was thinking if there is any way of returning a Flux<T> instead of a CompletionStage<List<T>> (Future)

Thank you.

**Comment by srinivasankavitha** [comment, marked as answer]
We depend directly on the DataLoader interface of`graphql-java`. Currently, only CompletableFuture is supported, so that would require changes in `graphql-java` to add the ability to return a Flux.
